libs/etterlatte-ktor/src/main/kotlin/token/BrukerTokenInfo.kt

```diff
@@ -83,6 +83,7 @@ data class HardkodaSystembruker private constructor(
         val uttrekk = HardkodaSystembruker(Systembrukere.UTTREKK)
         val statistikk = HardkodaSystembruker(Systembrukere.STATISTIKK)
         val tilgang = HardkodaSystembruker(Systembrukere.TILGANG)
+        val etteroppgjoer = HardkodaSystembruker(Systembrukere.ETTEROPPGJOER)
```

Comment:
Usikker på denne

Reply:
Jeg tenker det virker rimelig å ha en hardkoda bruker for etteroppgjør til å gjøre f.eks. dette oppslaget